client/blocks/comment-button/index.jsx

```diff
@@ -39,7 +45,7 @@ const CommentButton = React.createClass( {
 	render() {
 		let label;
 		const containerTag = this.props.tagName,
-			commentCount = this.props.commentCount;
+			commentCount = ( this.props.syncedCount === undefined ? this.props.commentCount : this.props.syncedCount );
```

Comment:
Shouldn’t we include the `syncedCount` prop in the `propTypes`? Actually, why do we need both `commentCount` and `syncedCount`?